pipeline.py

```diff
@@ -51,19 +57,23 @@ class CorpusManager:
     """
 
     def __init__(self, path_to_raw_txt_data: str):
-        pass
+        self.path_to_raw_text_data = path_to_raw_txt_data
+        self._storage = {}
+        self._scan_dataset()
 
     def _scan_dataset(self):
         """
         Register each dataset entry
         """
-        pass
+        for file in self.path_to_raw_text_data.glob("*_raw.txt"):
+            index = int(file.name[:-8])
```

Comment:
too straightforward. you'd better extract it with other means, for example regex groups

Reply:
noone will later understand where is this 8 from

Reply:
Fixed